core/main.go

```diff
@@ -11,31 +30,38 @@ import (
 	"time"
 
 	"github.com/edgexfoundry/edgex-go/pkg/clients/logging"
-	"github.com/hashicorp/vault/api"
 )
 
+var debug = false
 var lc = CreateLogging()
 
 // CreateLogging Logger functionality
 func CreateLogging() logger.LoggingClient {
-	return logger.NewClient(SecurityService, false, fmt.Sprintf("%s-%s.log", SecurityService, time.Now().Format("2006-01-02")), logger.InfoLog)
```

Comment:
This should not have been deleted, you may be building against an out-dated version of edgex-go locally, but the current master version of edgex-go needs this fourth argument.

Reply:
@anonymouse64 I'm not using master but delhi. The glide.yaml file I use has:
- package: github.com/edgexfoundry/edgex-go
  version: delhi
  subpackages:
  - pkg/clients/logging

The Golang import is: ``github.com/edgexfoundry/edgex-go/pkg/clients/logging``
And if I check the Logger code from Delhi:
https://github.com/edgexfoundry/edgex-go/blob/delhi/pkg/clients/logging/logger.go

It is indeed having ``NewClient(owningServiceName string, isRemote bool, logTarget string, logLevel string)`` with this 4th argument. Therefore I'm missing something, I did update the pkg with ``go get -u`` but I'm thinking the go build in the Makefile does not rebuild the dependent pkg... Can you help me out on this one?

Reply:
@anonymouse64 The -a option is used for the go build operation in the Makefile, rebuilding should happen. Should I also use the -i option?

Reply:
@anonymouse64 I got things solved. Had to specifically run:
``go get -v github.com/edgexfoundry/edgex-go/pkg/clients/logging``
And then I could use in my import clause:
``logger "github.com/edgexfoundry/edgex-go/pkg/clients/logging"``
Then the 4th argument was verified... 
Somehow I thought that the ``go get -u`` of the github.com/edgexfoundry/edgex-go would rebuild/reinstall ``./client/logging`` as it was underneath the same pkg structure.

Reply:
I think glide will do this for you if you just run `make prepare` before doing your build too. Glad you got it fixed. One question that I didn't notice though is why is this branch targeting Delhi? Is this work meant to be for a dot release or is it meant to go into Edinburgh?

Reply:
Both actually, it needed to be part of Delhi as it was an embarrassing behaviour (we could say a bug like Trevor experienced it). But it is also a stable foundation for Edinburgh now, I will sync up the master branch with it and continue the new features from there. We (hopefully) should be done with Delhi from now on.

Reply:
So which issues are being fixed for Delhi? There is more code in this PR than just fixes for the issues in the title of the PR. Also does this mean that we will have another dot release of delhi? With the versioning discussion that I think is going to the TSC if it hasn't already, security-services are supposed to be part of the larger "EdgeX" release, so if we do another release of the security services we need to do another release of everything else.

Reply:
* Add a LICENSE file: Fix #55
* vault-worker is unable to unlock vault if vault is killed with SIGKILL: Fix #54
* EdgeX upgrade Vault latest (Edinburgh Roadmap Item): Fix #53

I agree there is a lot of new/modified/deleted code but it's also totally subsequent to the many topics involved by the #54 fix. For example even if the Shamir Secret Sharing redesign was not in my plans originally, as I was fixing the init/unseal algorithms then it became obvious to adapt it, it's an essential part of the unseal issue (#54). It was awkward (code wise) to keep the old piece in the middle of the new one, specially as it was not technically making sense anymore. 
Using Vault REST API instead of the Vault Client was also a decision impacted by the rewrite entanglements, and adding more simplicity for future maintenances. 
In the commit message I did list the summary of the topics and beside the usual license related changes, and Vault upgrade, all is really pretty much about #54, but ramifications were important (TOML, config, algo, makefile, debug mode, etc.).
I honestly don't think we will have to bake new dot releases of Delhi anymore, besides blocking bugs I guess. From now on the new features will be on master branch and will target Edinburgh release.

Reply:
@anonymouse64 Would you be able now to approve the PR? Would like to close this one and when merged apply a resync to master to continue on Edinburgh features.